src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -268,6 +280,10 @@
     vertical-align: middle;
     background: transparent;
     -webkit-font-smoothing: antialiased;
+    -moz-font-smoothing: antialiased;
+    -o-font-smoothing: antialiased;
+    -ms-font-smoothing: antialiased;
+    font-smoothing: antialiased;
```

Comment:
Good question.  We have https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth in Firefox, but it's not really recommended.  Probably we should just revert this section of the patch, and leave it unchanged.